frontend/test/metabase/scenarios/sharing/reproductions/17657.cy.spec.js

```diff
@@ -0,0 +1,75 @@
+import { restore, sidebar } from "__support__/e2e/cypress";
+import { USERS } from "__support__/e2e/cypress_data";
+
+const {
+  admin: { first_name, last_name },
+} = USERS;
+
+describe("issue 17657", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    createSubscriptionWithoutRecipients();
+  });
+
+  it("frontend should gracefully handle the case of a subscription without a recipient (metabase#17657)", () => {
+    cy.visit("/dashboard/1");
+
+    cy.icon("share").click();
+    cy.findByText("Dashboard subscriptions").click();
+
+    cy.findByText(/^Emailed monthly/).click();
+
+    sidebar().within(() => {
+      cy.button("Done").should("be.disabled");
+    });
+
+    // Open the popover with all users
+    cy.findByPlaceholderText("Enter user names or email addresses").click();
+    // Pick admin as a recipient
+    cy.findByText(`${first_name} ${last_name}`).click();
+
+    sidebar().within(() => {
+      cy.button("Done").should("not.be.disabled");
+    });
+  });
+});
+
+function createSubscriptionWithoutRecipients() {
+  cy.request("POST", "/api/pulse", {
```

Comment:
There is also `cy.createPulse` command

Reply:
Thanks.
Tbh, I forgot to even look there :)